### PR TITLE
Modified front end to make cool div container for cool words generated

### PR DESCRIPTION
**Pull request by tmsreekanth98**
Modified the front end and added div containers for the names generated by the cool name api. Hover effect on mouse over also added.

Fixes Issue #2 

Screenshot:
coolapi

### CODE REVIEW

File: public/style.css [hunk 1]
```diff
@@ -9,6 +9,31 @@
...
 
+.listUI{
+  background: #101010;
+  color:white;
+  width:100%;
+  border: 1px solid black;
+  padding:5px;
+  margin: 15px;
+  border-radius:3px;
+  box-shadow: 2px 2px white !important;
```
Comment (mohitkyadav) [inline public/style.css]: use `uk-box-shadow`-* instead, using `!important` is not a good practice.
https://getuikit.com/docs/utility#box-shadow

File: public/style.css [hunk 2]
```diff
@@ -9,6 +9,31 @@
...
+
+.listUI:hover{
+  background: #151515;
+  color:white;
+  width:100%;
+  border: 1px solid black;
+  padding:5px;
+  margin: 15px;
+  border-radius:3px;
+  box-shadow: 2px 2px white !important;
```
Comment (mohitkyadav) [inline public/style.css]: same for hover, use `uk-box-shadow-hover-*`
https://getuikit.com/docs/utility#hover

File: public/style.css [hunk 3]
```diff
@@ -9,6 +9,31 @@
...
+  font-size:20px;
+  transform: scale(1.005);
```
Comment (mohitkyadav) [inline public/style.css]: Scaling transformation is not smooth, https://robots.thoughtbot.com/transitions-and-transforms

**Review by mohitkyadav — COMMENTED** [review]
Please see the comments

**Comment by tmsreekanth98** [comment]
Ok I will try it.

**Comment by tmsreekanth98** [comment]
Regarding the responsive table idea: I have done it, but if the name is too long the table has to be scrolled to the right horizontally.  The cool name generated has no space in between the word, so the text is not going to the next line and hence it is difficult to keep the table width in control as text spans across the width of the screen and not to the next line.

1 in each row will not have that problem. If u want i can send a PR for the 3 in a row version but it does not look good when names entered are big in size.

**Comment by mohitkyadav** [comment]
@tmsreekanth98 explore the `uk-width-*` in UIkit it allows to have dynamic width. That can be an alternative of using a table, I can understand that can be a problem since the number of columns is fixed so you can try it with 2 columns.

**Comment by tmsreekanth98** [comment]
Ok. I am looking into it. :)

**Comment by mohitkyadav** [comment]
Inactive

**Pull request closed without merging**